library/func_misc.py

```diff
@@ -37,6 +42,23 @@ def Call(self, args):
         return value.Null
 
 
+class Extend(vm._Callable):
+
+    def __init__(self):
+        # type: () -> None
+        pass
+
+    def Call(self, args):
+        # type: (typed_args.Reader) -> value_t
+
+        a = args.PosList()
+        b = args.PosList()
+        args.Done()
+
+        a.extend(b)
+        return value.List(a)
```

Comment:
Should return `value.Null` here . 

```
>>> a=[]
>>> print(a.extend([1,2,3]))
None
```

(similar discussion with mylist.reverse(), Aidan agreed)

Reply:
Ah, right! Nice catch